### PR TITLE
Small formatting fixes

### PR DESCRIPTION
**Pull request by IDoEverything**
# Summary
Went through every file and fixed small warnings

### CODE REVIEW

File: DSharpPlus/Clients/DiscordClient.cs [hunk 1]
```diff
@@ -784,9 +784,7 @@ internal DiscordThreadChannel InternalGetCachedThread(ulong threadId)
 
         internal DiscordChannel InternalGetCachedChannel(ulong channelId)
         {
-            DiscordDmChannel foundDmChannel = default;
-
-            if (this._privateChannels?.TryGetValue(channelId, out foundDmChannel) == true)
+            if (this._privateChannels?.TryGetValue(channelId, out var foundDmChannel) == true)
```
Comment (VelvetThePanda) [inline DSharpPlus/Clients/DiscordClient.cs:787]: this isn't JS; why are we doing `== true`

**Review by VelvetThePanda — CHANGES_REQUESTED** [review]
LGTM; one small nitpick

File: DSharpPlus/Clients/DiscordClient.cs (hunk 1 above)
Comment (Emzi0767) [reply DSharpPlus/Clients/DiscordClient.cs:787]: image

Comment (Emzi0767) [reply DSharpPlus/Clients/DiscordClient.cs:787]: The alternative to `left == true` is `left ?? false`

**Review by Emzi0767 — APPROVED** [review]

**Pull request merged**